ir/src/main/scala/org/scalajs/core/ir/Trees.scala

```diff
@@ -869,6 +869,37 @@ object Trees {
      */
     final case class Global(path: List[String]) extends JSNativeLoadSpec
 
+    /** Load from a module import.
+     *
+     *  The `module` is the ES module identifier. The `path` is a series of
+     *  nested property names starting from the module object.
+     *
+     *  The path can be empty, in which case the specification denotes the
+     *  module itself (whose exports are available as fields).
+     *
+     *  Any element in the path is a property selection from there. A module
+     *  import info with one path element is importing that particular value
+     *  from the module.
+     *
+     *  Examples:
+     *  {{{
+     *  // import { Bar as x } from 'foo'
+     *  Import("foo", List("Bar"))
+     *
+     *  // import { Bar as y } from 'foo'
+     *  // y.Baz
+     *  Import("foo", List("Bar", "Baz"))
+     *
+     *  // import * as x from 'foo' (whole module import)
+     *  Import("foo", Nil)
+     *
+     *  // import x from 'foo' (default import)
+     *  Import("foo", List("default"))
```

Comment:
> Also, it is actually quite unlikely for any legacy JS module to export a member named default, as it is a keyword in JS.

Good point. So let's emit the `_interopDefault` fallback.

> What syntax would you suggest for the * import, in that case?

For example

```scala
@JSImport("module.js", All)
```

where `All` is an object.